doc-downloader.functions.ps1

```diff
@@ -6,7 +6,7 @@ function CreateLogFile
 {param([string]$Dir)
     $CurrentTime = Get-Date -Format yyyy-MM-dd
     $filename = ("Eleos-" + ($CurrentTime + ".log"))
-    $filepath = ($DIR + $filename)
+    $filepath = ($DIR + "\" + $filename)
```

Comment:
Thanks for the patch @jscottSwift !!

Could I talk you into https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/join-path?view=powershell-7.3 for this, assuming it's supported across the relevant PowerShell versions?